sdk/dotnet/Makefile

```diff
@@ -49,8 +49,6 @@ install:: build install_plugin
 dotnet_test:: install
 	# include the version prefix/suffix to avoid generating a separate nupkg file
 	dotnet test /p:VersionPrefix=${VERSION_PREFIX} /p:VersionSuffix=${VERSION_SUFFIX}
-	# work around a bug where running `dotnet test` disables console stdinput display
-	stty echo
```

Comment:
Any reason why this was removed? This was added to workaround a bug where stdinput is all messed up after running the dotnet tests.

Reply:
yeah, it doesn't actually work. It's related to https://github.com/actions/runner/issues/241
I can wrap it in an if statement if needed, but I don't actually see this bug anymore so I think it's fixed in later versions of the dotnet sdk.

Reply:
Sounds good, we can always come back if we notice the issue after this gets merged.